agent/config/config.go

```diff
@@ -341,6 +344,16 @@ func environmentConfig() (Config, error) {
 		seelog.Debugf("Setting instance attribute %v: %v", attributeKey, attributeValue)
 	}
 
+	var additionalLocalRoutes []cnitypes.IPNet
+	additionalLocalRoutesEnv := os.Getenv("ECS_AWSVPC_ADDITIONAL_LOCAL_ROUTES")
+	if additionalLocalRoutesEnv != "" {
+		err := json.Unmarshal([]byte(additionalLocalRoutesEnv), &additionalLocalRoutes)
+		if err != nil {
+			seelog.Warnf("Invalid format for ECS_AWSVPC_ADDITIONAL_LOCAL_ROUTES, expected a json array of CIDRs: %v", err)
```

Comment:
same here, should we log the `additionalLocalRoutesEnv`?

Reply:
Yes, can do.

Reply:
I'm also going to leave this out.  We don't log the read environment variable for any of the other ones that we parse.  I'd rather leave this consistent for now and then revisit it later when we can change all of them.